examples/kubeflow-gke/README.md

```diff
@@ -0,0 +1,36 @@
+# Using `Fairing` with Kubeflow's TfJob inside a Jupyter Notebook
+
+This uses the appender builder and the jupyter-notebook service account to push and run a training job through Kubeflow's TfJob.
+
+## Requirements
+
+- Kubeflow on GKE
+
+## Instructions
+
+In the JupyterHub UI, use create a new instance of jupyter with `gcr.io/kubeflow-images-public/fairing:v0.0.1` as the image.
+
+This will only work on newer versions of Kubeflow. To use this on 0.3.x, you need to manually give `jupyter-notebook-role` `pods/log` permission.
+
+On 0.3.x, from this directory apply the pre-0.4-patch.yaml to the cluster with 
```

Comment:
I assume they could also modify their ksonnet config for Kubeflow and grant the appropriate permissions that way?

Reply:
I tried modifying my ksonnet config and that worked.